torch/autograd/_functions/tensor.py

```diff
@@ -121,6 +121,8 @@ class View(Function):
 
     @staticmethod
     def symbolic(g, i, sizes):
+        if len(sizes) == 1 and isinstance(sizes[0], torch.Size):
+            sizes = sizes[0]
         return g.op("Reshape", i, shape_i=sizes)
```

Comment:
I'm confused: `shape_i` implies that this is a single integer field, but sizes is always a tuple. Did this ever work?

Reply:
[It does](https://github.com/pytorch/pytorch/blob/master/torch/onnx.py#L138-L139). I'm only unsure why do we unpack the tuple if there's only one element. Does Reshape in ONNX requires you to change the type of an attribute if it's a single integer? That doesn't make sense to me

Reply:
The single element here is a `torch.Size` (which is a list of integers), not a single integer.

Reply:
Uh my bad. I misread the condition. Nevermind my comment...